docs/src/main/asciidoc/qute.adoc

```diff
@@ -26,45 +26,48 @@ Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {q
 
 The solution is located in the `qute-quickstart` link:{quickstarts-tree-url}/qute-quickstart[directory].
 
-== Hello World with Jakarta REST
-
-If you want to use Qute in your Jakarta REST application, you need to add an extension first:
+[[hello-qute-web]]
+== Hello World with Qute Web
 
-* either `quarkus-rest-qute` if you are using Quarkus REST (formerly RESTEasy Reactive):
-+
+If you want to use Qute in your Quarkus Web application, add the Qute Web extension first:
 [source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
 .pom.xml
 ----
 <dependency>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-rest-qute</artifactId>
+    <groupId>io.quarkiverse.qute.web</groupId>
```

Comment:
having docs in our main http docs  that uses extensions not part of platform - thats a first is it not?

Reply:
it is part of the platform

Reply:
![image](https://github.com/quarkusio/quarkus/assets/2223984/59c43767-8c54-4382-8d02-6d4cc6a5db6f)